docs/UserGuide.md

```diff
@@ -22,15 +22,15 @@
 ## Getting Started
 
 First, ensure that you have the necessary prerequisites:
-* **JDK `1.8.0_60`** or later
-* **git `2.14`** or later on the command line (run `git --version` in your OS terminal to confirm)
+* **Java 8** (JRE `1.8.0_60`) or later. You may download Java [here](https://www.java.com/en/).
+* **git `2.14`** or later on the command line (run `git --version` in your OS terminal to confirm). You may download git [here](https://git-scm.com/downloads).
 
 Next, download the latest executable Jar from our [releases](https://github.com/reposense/RepoSense/releases/latest).
 
 The simplest use case for RepoSense is to generate a report for the entire history of a repo. Here are the steps:
 1. Generate the report for the repo by executing the following command in a terminal:<br/>
-   Format : `java -jar RepoSense.jar --repo FULL_REPO_URL` (note the `.git` at the end)<br>
-   Example: `java -jar RepoSense.jar --repo https://github.com/reposense/RepoSense.git`
+   Format : `java -jar RepoSense.jar --repo FULL_REPO_URL --since d1` (note the `.git` at the end)<br>
+   Example: `java -jar RepoSense.jar --repo https://github.com/reposense/RepoSense.git --since d1`
```

Comment:
I think we shouldn't include `--since d1` here.

I believe Prof's wants to give first-time users a fast experience before they explore more complex args like `--since d1`.

Reply:
yes, perhaps add it under as notes or something

Reply:
Ok, made the changes.